docs/tools/editor/geany.md

```diff
@@ -2,93 +2,93 @@ author: xingjiapeng, MingqiHuang
 
 # Geany
 
-Geany 是一个轻量、便捷的编辑器，对于 Linux 环境下的初学者较为友好。
+Geany is a lightweight, easy-to-use text editor. It is friendly for beginners in Linux environment.
 
-与 Dev-C++ 一样，它可以编译运行单个文件。
+Like Dev-C++, it supports single file compilation.
 
-不过，它可以在 Linux/Windows/macOS 下运行。
+However, it is compatible in Linux, Windows and macOS.
 
-其官网为：<https://geany.org/>
+The homepage of Geany is <https://geany.org/>
```

Comment:
[Geany](https://geany.org) is a lightweight, easy-to-use text editor friendly for beginners in Linux environment. Like Dev-C++, it supports single file compilation but is compatible in Linux, Windows and macOS.